packages/greencheck/composer.json

```diff
@@ -17,13 +17,14 @@
         "gedmo/doctrine-extensions": "~2.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "3.7.*",
-        "doctrine/data-fixtures": "dev-master",
-        "friendsofphp/php-cs-fixer": "^2.14",
-        "symfony/console": "^4.2"
+        "phpunit/phpunit": "*",
```

Comment:
best to pin it to a certain version, for example ^8.0 so it won't break if phpunit 9 is released which might have BC breaks